src/ludamus/templates/panel/proposals.html

```
@@ -72,6 +72,39 @@
+                                    name="category"
+                                    class="filter-input w-full px-3 py-2 border border-border rounded-lg text-sm truncate">
+                                <option value="">{% translate "All categories" %}</option>
+                                {% for c in categories %}
+                                    <option value="{{ c.pk }}"
+                                            {% if filter_category_pk == c.pk %}selected{% endif %}>{{ c.name }}</option>
+                                {% endfor %}
+                            </select>
+                        </div>
+                    {% endif %}
+                    <div class="w-full sm:w-48 min-w-0">
+                        <label for="filter-status"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)